.github/workflows/run-synthetics.yml

```diff
@@ -37,12 +37,12 @@ jobs:
         with:
           node-version: '12.22.3'
       - run: npm install
-      - run: "SYNTHETICS_JUNIT_FILE='junit-synthetics.xml' npx @elastic/synthetics . --reporter=junit --suite-params='{\"url\": \"https://elastic.github.io/synthetics-demo/\"} '"
-        continue-on-error: true
+      # --quiet-exit-code makes synthetics always exit with a code of 0 even if tests fail so the Junit reporter can run
+      - run: "SYNTHETICS_JUNIT_FILE='junit-synthetics.xml' npx @elastic/synthetics . --reporter=junit --quiet-exit-code --params='{\"url\": \"https://elastic.github.io/synthetics-demo/\"} '"
```

Comment:
you don't need to pass params right? It should work as we have `synthetics.config.ts`. Just need to pass correct env.